src/features/HomePage/index.tsx

```diff
@@ -36,9 +45,11 @@ const HomePage = () => {
       <MiddleContainer>
         <LeftMiddleContainer>
           {hasUnreadMessages ? <NewMessages /> : <Welcome />}
-          <Announcements />
+          {!isMentor && <Announcements />}
+          {isMentor && mentor && <ProfileWidget mentor={mentor} />}
         </LeftMiddleContainer>
         <RightMiddleContainer>
+          {isMentor && <Announcements />}
```

Comment:
We dont need to define `isMentor`.
If we can select a mentor with the users id, then we know it is mentor. 

So basically we can just 
```typescript
mentor && <Component />
```